### ISSUE TITLE
defineSearchAction not defined in nuxt

### ISSUE THREAD

**Issue by LouisHaftmann** [vueuse/schema-org#25]
`defineSearchAction` is not in auto imports and importing from `@vueuse/schema-org/runtime` doesn't work

**Comment by joshwcorbett** [comment]
I experienced this as well, explicitly importing seems to do fine for local development and production build but breaks preview build on vercel.
```ts
// this works but I hate it
import { defineSearchAction } from '@vueuse/schema-org/runtime/provider'
```

Local preview and build works ok, but vercel says otherwise regarding the preview build, prod build is fine tho.

<img width="1243" alt="Screen Shot 2022-09-15 at 7 39 01 AM" src="https://user-images.githubusercontent.com/23065423/190418757-1be8d515-466a-45b7-ac67-6db147bf3f8f.png">

**Comment by harlan-zw** [comment]
Thanks for flagging this, seems fixed in [v1.1.0-beta.3](https://github.com/vueuse/schema-org/releases/tag/v1.1.0-beta.3)

(see https://stackblitz.com/edit/nuxt-starter-z9np1t?file=package.json,pages%2Findex.vue)

If you still have that error when updating, would you mind making a new issue @joshwcorbett? Seems like it could be unrelated
